src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -187,14 +194,86 @@ def get_current_exposure(self, index: pd.Index) -> pd.Series:
                 return pd.Series(exposure, index=index)
 
 
+class WastingDiseaseState(DiseaseState):
```

Comment:
Can you actually refactor `DiseaseState` so this subclassing is easier? Let's talk about it.